src/components/utdragFraSykefravaeret/UtdragFraSykefravaeret.tsx

```diff
@@ -141,16 +139,25 @@ export const Sykmeldinger = ({
   );
 
   return (
-    <div className={"mb-10"}>
+    <div className="mb-10 [&>*]:mb-4">
```

Comment:
Nice - er dette sånn man gjør children selecting i tailwind?

Reply:
Ja, ser sånn ut! Har ikke eksperimentert med `[&>*:not(last-child)]` type greier enda da.